pkg/sensors/sensor_types_windows.go

```diff
@@ -0,0 +1,24 @@
+// SPDX-License-Identifier: Apache-2.0
```

Comment:
nit, could you plz rename rename 'sensor' to 'sensors' in the path, so we'd have:

```
pkg/sensors/sensors_types_linux.go      
pkg/sensors/sensors_types_windows.go    
```